### PR TITLE
Jormungandr: night_bus_filter_max_factor is a float

### PR DESCRIPTION
**Pull request by pbougue**
JIRA: http://jira.canaltp.fr/browse/NAVITIAII-2056

### CODE REVIEW

File: source/jormungandr/jormungandr/interfaces/v1/Journeys.py [hunk 1]
```diff
@@ -577,7 +577,7 @@ def __init__(self):
 
         parser_get.add_argument("_walking_transfer_penalty", type=int)
         parser_get.add_argument("_night_bus_filter_base_factor", type=int)
-        parser_get.add_argument("_night_bus_filter_max_factor", type=int)
+        parser_get.add_argument("_night_bus_filter_max_factor", type=float_gt_0)
```
Comment (TeXitoi) [inline source/jormungandr/jormungandr/interfaces/v1/Journeys.py]: We should manage a 0 value. A test can be great.

**Comment by TeXitoi** [comment]
OK for me.

File: source/jormungandr/tests/routing_tests.py [hunk 2]
```diff
@@ -307,6 +307,34 @@ def test_journeys_wheelchair_profile(self):
...
+
+        query = "journeys?from={from_coord}&to={to_coord}&datetime={d}&" \
+                         "_night_bus_filter_max_factor={_night_bus_filter_max_factor}"\
+            .format(from_coord=s_coord, to_coord=r_coord, d="20120614T08",
+                    _night_bus_filter_max_factor=2.8)
+
+        response = self.query_region(query)
+        is_valid_journey_response(response, self.tester, query)
+
+        #and the second should be 0 initialized
```
Comment (antoine-de) [inline source/jormungandr/tests/routing_tests.py]: I don't understand this comment

File: source/jormungandr/tests/routing_tests.py [hunk 3]
```diff
@@ -307,6 +307,34 @@ def test_journeys_wheelchair_profile(self):
...
+        journeys = get_not_null(response, "journeys")
+        assert journeys[0]["requested_date_time"] == "20120614T080000"
```
Comment (antoine-de) [inline source/jormungandr/tests/routing_tests.py]: I don't also understand this assert, what do you want to check ?

**Pull request merged**